### PR TITLE
Add initial core project contacts

### PR DESCRIPTION
**Pull request by jarrodmillman**
@QuLogic @rossbar @melissawm @jorisvandenbossche @jni @rgommers @Carreau Please approve this if you are prepared to do the following:

0. Find "your core project" by finding the changed file containing your handle.
1. Once the draft site is  deployed (around 5/17), share the site with "your core project".
2. Ask "your core project" to review the SPEC process documents paying close attention to the Core Project document.
3. Ask "your core project" whether they are willing to be a Core Project in the SPEC process.
4. If  "your core project" agrees, make a PR changing `draft: true` to ` draft: false` in "your core project" file.

Once we set the `draft: false`  for all the initial Core Projects, we will talk more about what this role entails and we will add people to the `contact:` fields for the Core Projects.  Basically, the main responsibility is to be a conduit between the SPECs and "your core project". Letting the Core Projects know about relevant SPECs, and helping ensure Core Project endorsements are correct.

### CODE REVIEW

**Review by rossbar — APPROVED** [review]
Approving to indicate my willingness to serve as "liaison" to networkx for the core-project procedure.

**Comment by bsipocz** [comment]
If you feel it's generic enough library, astropy can be also be added to the list, and am willing to be the "liaison".

**Comment by jarrodmillman** [comment]
Astropy may be a good candidate for inclusion.  We would need to check that it meets the characteristics of a Core Project.  (I suspect it does, but I am not familiar with its governance structure for instance.  My hope is to ensure that among other things that all the Core Projects are community run.)  We would also need to make sure it matches up with this statement:

> Core Projects are depended upon by most of the other projects, and often provide basic data structures, drawing primitives, or implementations of fundamental algorithms. Due to their central position in the ecosystem, the policies, practices, and tooling used by the Core Projects are widely seen by the ecosystem and impact many other projects.

I have a **slight** interest in starting small (I worry that starting with too many people and projects may make starting more difficult and the list of people and projects is already somewhat large); but, I also want to get a lot of people engaged quickly.  So I see an argument either way.

My initial thought was to limit the Core Projects to the current list (in this PR) until we get closer to announcing things at the SciPy conference in July.  Does that make sense to everyone?  Or should we consider adding astropy (and perhaps a few other projects) to the list of Core Projects now?

Ultimately, I want to keep the list of Core Projects small b/c it is only supposed to represent the most widely used projects in the ecosystem.  Also if we add many more Core Projects, we may want to revisit the number of Core Projects required before a SPEC is considered "Endorsed".  We currently require two Core Projects to sign on.  But, 2/8 is more meaningful to me than 2/20.  So I suspect adding more Core Projects may make moving SPECs to the endorse stage more difficult.

So my initial inclination is to limit the initial list of Core Projects to these 8 projects (of course, 9 isn't much different than 8).  And then before the SciPy conference, the steering committee should review the criteria for Core Projects and consider whether or not to expand the list.  And, if we decide to expand the list, we need to decide on the list of projects to include/recruit.

**Comment by jarrodmillman** [comment]
BTW, we can add additional `adopters` to the Core Projects when they "sign on."  Moreover, the list of `adopters` doesn't need to be limited to steering committee members.  The list of `adopters` also doesn't need to be limited to the Core Projects' core developer team.  The main requirement is to be familiar enough with the project that you are on the projects main communication forums, know how they make decisions, and are able to tell when the project has agreed to something.  We could make this more formal (if needed), but for now we are a small enough community that we shouldn't worry too much about completely formalizing things at the start.

**Comment by bsipocz** [comment]
> My initial thought was to limit the Core Projects to the current list (in this PR) until we get closer to announcing things at the SciPy conference in July. Does that make sense to everyone? Or should we consider adding astropy (and perhaps a few other projects) to the list of Core Projects now?

Sounds totally reasonable to wait until scipy. 
With the community-run criteria I would also wait until elections are done in the project (maybe sometimes in the next year knowing the reluctance around it) as the current governance is self-appointed and very much run in the worst BDFL style.

**Review by QuLogic — APPROVED** [review]

**Comment by rgommers** [comment]
I'd recommend waiting. Astropy is a large and important project, but the same time also very niche. I'd personaly draw the line at things both widely used and not specific to a single domain. Probably at about the scikit-image/scikit-learn/statsmodels/PyMC3 level.

**Review by rgommers — APPROVED** [review]

**Comment by bsipocz** [comment]
I like that definition of core projects have to be widely used and not single domain-specific. 
Maybe it's worth adding it to the list of definition? If I read it correctly it only details out the wide usage, community dev, etc.

File: core-projects/pandas.md
Comment (jorisvandenbossche) [inline core-projects/pandas.md:9]:
```suggestion
license: https://github.com/pandas-dev/pandas/blob/master/LICENSE
```

**Review by jorisvandenbossche — APPROVED** [review]

**Comment by jarrodmillman** [comment]
@Carreau I just realized the IPython file isn't part of this PR.  Regardless, could you review this file (and if you are fine the items outlined in the description above) approve this PR:
https://raw.githubusercontent.com/scientific-python/specs/main/core-projects/ipython.md

**Comment by jni** [comment]
Sorry I'm suuuuper confused by the "adopters" name. Should it be taken to mean "people in that project responsible for driving adoption of specific specs"? To me a more appropriate term might be "delegates"? I'm also confused about making individuals responsible, rather than e.g. `skimage-core@python.org`, which makes much more sense since the idea is that these projects have an established governance etc. that does not come down to individuals.

**Review by jni — APPROVED** [review]

File: core-projects/matplotlib.md
Comment (munkm) [inline core-projects/matplotlib.md:9]: maybe this license link shouldn't point to a specific version? e.g. 

```suggestion
license: https://matplotlib.org/stable/users/license.html
```

**Review by munkm — APPROVED** [review]

**Comment by bsipocz** [comment]
> We need someone to volunteer to communicate with the scikit-learn team

I can volunteer for that, at least until a scikit-learn maintainer joins the SC

**Review by Carreau — APPROVED** [review]

**Comment by jarrodmillman** [comment]
@bsipocz Thanks!  I added your name as the contact for scikit-learn.  Would you mind approving this PR to confirm you are wiling to make contact with the scikit-learn team regarding being a Core Project for the SPEC process?

Stefan and I have already spoken to @ogrisel and @GaelVaroquaux, but we haven't reached out to the larger developer community yet.  I am not even 100% sure where the best place to post is (e.g., should you create an issue or send an email to their mailing list...)

At any rate, Stefan and I are happy to join in the discussion so please feel free to cc us or @mention us.  (That goes for any of the Core Project contacts.)  The initial ask of the Core Projects is pretty minimal.  They don't need to adopt or endorse anything.  The main thing we want is for the Core Projects to agree to participate to the extent that it makes sense for them.

**Review by bsipocz — APPROVED** [review]

**Comment by stefanv** [comment]
> Sorry I'm suuuuper confused by the "adopters" name. Should it be taken to mean "people in that project responsible for driving adoption of specific specs"? To me a more appropriate term might be "delegates"? I'm also confused about making individuals responsible, rather than e.g. `skimage-core@python.org`, which makes much more sense since the idea is that these projects have an established governance etc. that does not come down to individuals.

Jarrod clarified the naming.

**Pull request merged**
